### ISSUE TITLE
App-Shortcut to certificate screen

### ISSUE THREAD

**Issue by jeffbs** [corona-warn-app/cwa-app-android#4474]
In the current situation you have to show your vacation certificate quite often. It would be handy to have an app-shortcut to quickly navigate to the list of certificate or even directly to the latest.

**Comment by Ein-Tim** [comment]
Hi @jeffbs 👋

Thanks for the suggestion! I would suggest you take a look at https://github.com/corona-warn-app/cwa-wishlist/issues/504, which is basically the same request as yours, or?

Thanks and have a nice Sunday!

**Comment by jeffbs** [comment]
yeah, thats what i want. :)

**Comment by Ein-Tim** [comment]
@jeffbs Okay, then I suggest to monitor https://github.com/corona-warn-app/cwa-wishlist/issues/504 and close this issue (:

**Comment by jeffbs** [comment]
> @jeffbs Okay, then I suggest to monitor https://github.com/corona-warn-app/cwa-wishlist/issues/504 and close this issue (: 

just edited my latest comment.

**Comment by Ein-Tim** [comment]
@jeffbs 

Yes, this issue is quite old, as I'm also only a community member I can't comment on why this is not realized. I guess other features were prioritized. 

Even though the issue is old, I suggest you ask for an update in https://github.com/corona-warn-app/cwa-wishlist/issues/504 and close this issue. (-:

**Comment by Ein-Tim** [comment]
@jeffbs 

Would you like to close this issue in favor of https://github.com/corona-warn-app/cwa-wishlist/issues/504?

**Comment by svengabr** [comment]
Duplicate of https://github.com/corona-warn-app/cwa-wishlist/issues/504

**Comment by svengabr** [comment]
Hello everyone, I know the ticket https://github.com/corona-warn-app/cwa-wishlist/issues/504 is quite old however I did just reboot the discussion and hoping to get feedback in Jira in the near future.

This ticket is marked as duplicate and will be closed in favor of https://github.com/corona-warn-app/cwa-wishlist/issues/504
